lib/application_insights/channel/contracts/envelope.rb

```diff
@@ -1,266 +1,40 @@
-          if value == @defaults['os']
-            @values.delete 'os' if @values.key? 'os'
-          else
-            @values['os'] = value
-          end
-        end
-        
-        # Gets the os_ver property.
-        def os_ver
-          @values.fetch('osVer') { 
-            @values['osVer'] = nil
-          }
-        end
-        
-        # Sets the os_ver property.
-        def os_ver=(value)
-          if value == @defaults['osVer']
-            @values.delete 'osVer' if @values.key? 'osVer'
-          else
-            @values['osVer'] = value
-          end
-        end
-        
-        # Gets the app_id property.
-        def app_id
-          @values.fetch('appId') { 
-            @values['appId'] = nil
-          }
-        end
-        
-        # Sets the app_id property.
-        def app_id=(value)
-          if value == @defaults['appId']
-            @values.delete 'appId' if @values.key? 'appId'
-          else
-            @values['appId'] = value
-          end
-        end
-        
-        # Gets the app_ver property.
-        def app_ver
-          @values.fetch('appVer') { 
-            @values['appVer'] = nil
-          }
-        end
-        
-        # Sets the app_ver property.
-        def app_ver=(value)
-          if value == @defaults['appVer']
-            @values.delete 'appVer' if @values.key? 'appVer'
-          else
-            @values['appVer'] = value
-          end
-        end
-        
-        # Gets the user_id property.
-        def user_id
-          @values.fetch('userId') { 
-            @values['userId'] = nil
-          }
-        end
-        
-        # Sets the user_id property.
-        def user_id=(value)
-          if value == @defaults['userId']
-            @values.delete 'userId' if @values.key? 'userId'
-          else
-            @values['userId'] = value
-          end
-        end
-        
-        # Gets the tags property.
-        def tags
-          @values.fetch('tags') { 
-            @values['tags'] = {}
-          }
-        end
-        
-        # Sets the tags property.
-        def tags=(value)
-          if value == @defaults['tags']
-            @values.delete 'tags' if @values.key? 'tags'
-          else
-            @values['tags'] = value
-          end
-        end
-        
-        # Gets the data property.
-        def data
-          @values.fetch('data') { 
-            @values['data'] = nil
-          }
-        end
-        
-        # Sets the data property.
-        def data=(value)
-          if value == @defaults['data']
-            @values.delete 'data' if @values.key? 'data'
-          else
-            @values['data'] = value
-          end
-        end
-      end
+module ApplicationInsights::Channel::Contracts
+  class Envelope
+    include JsonSerializable
+
+    attr_accessor :ver, :name, :time, :sample_rate, :seq, :i_key, :flags,
+      :device_id, :os, :os_ver, :app_id, :app_ver, :user_id, :tags, :data
+
+    attribute_mapping(
```

Comment:
The schema had fields defined in order. Could you please reorder the fields same as before? Same to other contract classes.